packages/fluent-ui/src/DateWidget/DateWidget.tsx

```diff
@@ -86,18 +85,18 @@ const parseDate = (dateStr?: string) => {
 const DateWidget = ({
   id,
   required,
-  readonly,
-  disabled,
+  // readonly,
+  // disabled,
```

Comment:
Delete this maybe since it is not used... or perhaps you meant to uncomment?

Reply:
```suggestion
```